### ISSUE TITLE
movie: add option to ignore if sub-directory with frame images exists

### ISSUE THREAD

**Issue by anbj** [GenericMappingTools/gmt#6665]
In ``movie``; if the sub-directory with frame images exists, ``movie`` will exit immediately:

```
[...]
$ bash movie.sh 
movie [INFORMATION]: No width given for progress indicator F. Setting width to 13.8c.
movie [INFORMATION]: FFmpeg 4.3.4-0+deb11u1 found.
[...]
movie [ERROR]: Working directory /home/anbj/Downloads/oga/uk already exist and -Z was not specified - exiting
[...]
```

* What about adding a modifier to ``-Z`` that tells ``gmt`` to ignore if the *prefix* folder already exists - just overwrite (or delete?) and proceed.
* The error *[..] already exist and -Z was not specified - exiting*, implies to me that specifying ``-Z`` *should* do this. The command invoked does contain ``-Z`` so this is a bit confusing (though this would apply if the script would run to the end and finish).
* Why? This problem arises due to the iterative nature of making movies; testing and failing and lots of `^C`-'ing - which often results in old waste lying around in the *prefix* folder.

**Comment by PaulWessel** [comment]
I agree that if **-Z** is set then we should just zap that directory, no questions asked.  I am tired of that dance myself.

**Comment by anbj** [comment]
Fixed in #6666.
